libraries/r36-buildtime/context/build

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
```

Comment:
I think this can be simplified a bit. We don't have the shared cache right now and we can install dependencies in the normal location so I think we don't have to do the copying and can just make 1 call to rip

Reply:
for sure i just wanted to make this PR a refactor for the pieces that didn't need to change

Reply:
The point of migrating to IPA is to get a chance to fix/improve the way we do things. Bringing over this old cruft to the master branch doesn't seem right to me. Having to install things once should also reduce build times drastically for R